src/client/dfs/SConscript

```diff
@@ -14,28 +14,31 @@ def scons():
                 if 'ID' in line:
                     line = line.strip()
                     _, distro = line.strip().split('=')
+                    distro = distro.strip('"')
                     break
-        if distro in ['"centos"', '"rhel"', '"fedora"']:
+        if distro in ['centos', 'rhel', 'fedora', 'opensuse']:
             import rpm
+            from pkg_resources import parse_version
+
             ts = rpm.TransactionSet()
             headers = ts.dbMatch('name', 'lustre')
             if not headers:
-                print("No package lustre found.")
+                headers = ts.dbMatch('name', 'lustre-client')
+            if headers and parse_version(headers['version']) >= parse_version('2.12.57'):
```

Comment:
One possible alternative here is to use SCons configure with perhaps a custom header that fails to compile if the check fails

config = Configure(env)
has_support = config.CheckHeader(header_name)
config.Finish()

Reply:
> One possible alternative here is to use SCons configure with perhaps a custom header that fails to compile if the check fails
> 
> config = Configure(env)
> has_support = config.CheckHeader(header_name)
> config.Finish()

Thanks for the hint Jeff, but by header do you mean an include file name ?